
syntax = "proto2";

package mesop.components.select;

message SelectChangeEvent {
    repeated string values = 1;
}

message SelectType {
  optional bool disabled = 2;
  optional bool disable_ripple = 3;
  optional bool multiple = 18;
  optional double tab_index = 4;
  optional string placeholder = 6;
  optional string value = 9;
  optional string on_select_opened_change_event_handler_id = 14;
  optional string on_select_selection_change_event_handler_id = 15;

 repeated SelectOption options = 16;
  optional string label = 17;
}

message SelectOption {
  optional string label = 1;
  optional string value = 2;
}
